### PR TITLE
Master project template tde

### PR DESCRIPTION
**Pull request by tde-banana-odoo**
Task: https://www.odoo.com/web?debug=#id=21320&view_type=form&model=project.task&action=333&active_id=248&menu_id=4720

### CODE REVIEW

**Comment by tde-banana-odoo** [comment]
@jem-odoo Could you have a look ?

File: addons/mail/models/mail_thread.py [hunk 1]
```diff
@@ -405,8 +405,7 @@ def _get_tracked_fields(self, updated_fields):
     def _track_subtype(self, init_values):
         """ Give the subtypes triggered by the changes on the record according
         to values that have been updated.
-
-        :param ids: list of a single ID, the ID of the record being modified
+         :param ids: list of a single ID, the ID of the record being modified
```
Comment (tivisse) [inline addons/mail/models/mail_thread.py]: useless diff

File: addons/mail/views/mail_templates.xml [hunk 2]
```diff
@@ -23,5 +23,10 @@
                 <link rel="stylesheet" href="/mail/static/src/less/thread.less" type="text/less"/>
             </xpath>
         </template>
+
+        <template id="mail_data_template_user_notification" name="mail_data_template_user_notification">
+<p>Dear ${object.user_id.name}</p>
+<p>You suck.</p>
```
Comment (tivisse) [inline addons/mail/views/mail_templates.xml]: I hope you'll get more imagination before merging this ;)

Comment (bst-odoo) [inline addons/mail/views/mail_templates.xml]: Wait what ?

**Pull request merged**